docs/content/docs/test/Toast.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Testing toasts
+type: docs
+---
+
+# Testing toasts
+
+[android.widget.Toast](https://developer.android.com/reference/android/widget/Toast)
+
+{{< hint warning>}}
+Testing toasts reliably is a hard task because some other toast can show up and yours will be queued. 
+We hack app with proxy object to mitigate this.
```

Comment:
```suggestion
Testing toasts is a hard because other toasts can show up and yours will be queued. 
We hack app with proxy object to mitigate this.
```